### ISSUE TITLE
Icons not updating

### ISSUE THREAD

**Issue by annomatik** [XBMC-Addons/resource.images.studios.white#61]
Hi! Current version is .32, right? https://mirrors.kodi.tv/addons/matrix/resource.images.studios.white/ only contains 0.30.

That's why it does not update, right? Thanks!

**Comment by AMoo-Miki** [comment]
I have a feeling not having new releases could have something to do with TexturePacker not being able to handle large packs like this.

**Comment by ronie** [comment]
nah, i haven't submitted an update to the addon repo in quite some time.
i'll have a look at it.

**Comment by psonnosp** [comment]
It's still on 0.30 so what happened to this?

Managed to download it here though so no problem but I guess many others doesn't.

These were still missing for me though even after updating so maybe you want to add them.
[resource.images.studios.white.zip](https://github.com/user-attachments/files/26639659/resource.images.studios.white.zip)

**Comment by ronie** [comment]
@psonnosp thanx for the contribution, i've added your icons. https://github.com/XBMC-Addons/resource.images.studios.white/commit/0eb3a095f9c038f9c6079b0d8f5a2082b495b2c6

i will update this addon (long long overdue) in the kodi addon repo as well.
